src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/search.js

```diff
@@ -205,7 +213,15 @@ require(["fuse"], function (Fuse) {
         break;
 
       default:
-        searchingForResultsBanner();
+        // if environment variable "FUSE_ACTIVE" is set to true
+        if (
+          document.documentElement.getAttribute("data-fuse_active") === "true"
```

Comment:
Is this attribute set only once Fuse.js has loaded?

Reply:
yes, it is just not to show the bar `searching` for local pages. It will be set only if fuse got initialize properly